project/AppDependencies.scala

```diff
@@ -21,7 +21,7 @@ object AppDependencies {
   )
```

Comment:
line 10, does this not need changing to "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"      % "[latest version]"
as per https://github.com/hmrc/hmrc-mongo/blob/main/MIGRATION.md

Reply:
I don't think so, because its using "org.reactivemongo" not "uk.gov.hmrc.mongo"